source/scripting_v3/GTA/Entities/Entity.cs

```diff
@@ -561,11 +584,53 @@ public Vector3 Velocity
 		}
 
 		/// <summary>
-		/// Gets the rotation velocity of this <see cref="Entity"/>.
+		/// Gets or sets the rotation velocity of this <see cref="Entity"/> in local axes.
 		/// </summary>
 		public Vector3 RotationVelocity
 		{
 			get => Function.Call<Vector3>(Hash.GET_ENTITY_ROTATION_VELOCITY, Handle);
+			set
+			{
+				var address = MemoryAddress;
+				if (address == IntPtr.Zero)
+				{
+					return;
+				}
+
+				var angularVelocityInLocalAxes = Quaternion * value;
+				SHVDN.NativeMemory.SetEntityAngularVelocity(address, angularVelocityInLocalAxes.X, angularVelocityInLocalAxes.Y, angularVelocityInLocalAxes.Z);
+			}
+		}
+
+		/// <summary>
+		/// Gets or sets the rotation velocity of this <see cref="Entity"/> in world axes.
+		/// </summary>
+		public Vector3 RotationVelocityWorldAxis
```

Comment:
I'd call this `WorldRotationVelocity` for better consistency with e.g. `DefineAttachedObjectHelper.WorldPos` or `Vector3.WorldUp`. And:

[...] in world space.